eis_qgis_plugin/processing/algorithms/pca.py

```diff
@@ -19,7 +19,7 @@ def __init__(self) -> None:
 
     def initAlgorithm(self, config=None):
 
-        self.alg_parameters = ["input_raster", "components", "output_folder"]
+        self.alg_parameters = ["input_raster", "components", "output_file"]
 
         self.addParameter(
             QgsProcessingParameterRasterLayer(
```

Comment:
I am not sure but the input might be vector layer (at least toolkit takes now geodataframe)